### ISSUE TITLE
SmartIR not working with HomeKit on hassio 0.109.0

### ISSUE THREAD

**Issue by spartanunitato** [smartHomeHub/SmartIR#330]
SmartIR causes all my HomeKit devices to show up as not responding, and issue is only resolved when I remove all my smartIR devices from HomeKit. This issue only appeared after I updated to 0.109.0 this morning.

**Comment by vassilis-panos** [comment]
Since I'm not using homekit, any error logs would be helpful.

**Comment by litinoveweedle** [comment]
Probably nor related, but I found this in my Hassio log after upgrade to 0.109.0

 Log Details (WARNING)

```
Logger: homeassistant.util.async_
Source: util/async_.py:120
First occurred: 1:11:32 AM (1 occurrences)
Last logged: 1:11:32 AM
Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for smartir doing I/O at custom_components/smartir/__init__.py, line 68: request = requests.get(MANIFEST_URL.format(branch), stream=True, timeout=10) 
```

**Comment by spartanunitato** [comment]
@vassilis-panos The issue is fixed after I updated to hassio 0.109.1! It was a hassio problem not smartir.
